code/modules/mob/living/carbon/alien/adult/alien_powers.dm

```diff
@@ -338,32 +338,31 @@ Doesn't work on other aliens/AI.*/
 	new choice_path(owner.loc)
 	return TRUE
 
-/datum/action/cooldown/alien/sneak
+/datum/action/cooldown/sneak/alien
 	name = "Sneak"
 	desc = "Blend into the shadows to stalk your prey."
-	button_icon_state = "alien_sneak"
+	button_icon_state = "sneak_alien"
```

Comment:
This seems like a mistake, don't change the icon state of this unless you're changing it in the `dmi` file itself

Reply:
yeah big mistake